source/setup.py

```diff
@@ -110,80 +92,60 @@ def getRecursiveDataFiles(dest,source,excludes=()):
 	[rulesList.extend(getRecursiveDataFiles(os.path.join(dest,dirName),os.path.join(source,dirName),excludes=excludes)) for dirName in os.listdir(source) if os.path.isdir(os.path.join(source,dirName)) and not dirName.startswith('.')]
 	return rulesList
 
-setup(
-	name = name,
-	version=version,
-	description=description,
-	url=url,
-	classifiers=[
-'Development Status :: 3 - Alpha',
-'Environment :: Win32 (MS Windows)',
-'Topic :: Adaptive Technologies'
-'Intended Audience :: Developers',
-'Intended Audience :: End Users/Desktop',
-'License :: OSI Approved :: GNU General Public License (GPL)',
-'Natural Language :: English',
-'Programming Language :: Python',
-'Operating System :: Microsoft :: Windows',
-],
-	cmdclass={"py2exe": py2exe},
-	windows=[
-		{
-			"script":"nvda.pyw",
-			"dest_base":"nvda_noUIAccess",
-			"uiAccess": False,
-			"icon_resources":[(1,"images/nvda.ico")],
-			"other_resources": [], # Populated at run time
-			"version":formatBuildVersionString(),
-			"description":"NVDA application",
-			"product_name":name,
-			"product_version":version,
-			"copyright": NVDAcopyright,
-			"company_name":publisher,
-		},
-		# The nvda_uiAccess target will be added at runtime if required.
-		{
-			"script": "nvda_slave.pyw",
-			"uiAccess": False,
-			"icon_resources": [(1,"images/nvda.ico")],
-			"other_resources": [], # Populated at run time
-			"version":formatBuildVersionString(),
-			"description": name,
-			"product_name":name,
-			"product_version": version,
-			"copyright": NVDAcopyright,
-			"company_name": publisher,
-		},
-		{
-			"script": "nvda_eoaProxy.pyw",
-			# uiAccess will be enabled at runtime if appropriate.
-			"uiAccess": False,
-			"icon_resources": [(1,"images/nvda.ico")],
-			"other_resources": [], # Populated at run time
-			"version":formatBuildVersionString(),
-			"description": "NVDA Ease of Access proxy",
-			"product_name":name,
-			"product_version": version,
-			"copyright": NVDAcopyright,
-			"company_name": publisher,
-		},
-	],
+
+def _genManifestTemplate(shouldHaveUIAccess: bool) -> Tuple[int, int, bytes]:

```

Comment:
Since we are on Python 3.11 where standard collections  can be type annotated this can be simplified to:
```suggestion
def _genManifestTemplate(shouldHaveUIAccess: bool) -> tuple[int, int, bytes]:
```